setup.py

```diff
@@ -19,6 +19,7 @@
     name="hf-doc-builder",
     version="0.5.0.dev0",
     author="Hugging Face, Inc.",
+    license="Apache",
     author_email="sylvain@huggingface.co",
```

Comment:
@LysandreJik who should we put for this now?

Reply:
mishig's email? wdyt @mishig25 ?

Reply:
docs@huggingface.co would be good